CodeStarterCamp_Week2/Step3.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Step3.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 고혜지 on 1/3/24.
+//
+
+import Foundation
+
+class LottoManager {
+  
+  private var winningRecords: Dictionary<String, Set<Int>> = .init()
```

Comment:
```suggestion
  private var winningRecords: [String: Set<Int>] = .init()
```
요즘은 <> 안에 안쓰고 요렇게 쓰는게 국룰인거 같더라구요 ㅎㅎ

Reply:
이런 팁 너무 좋아요ㅎㅎ